docs/transfer-setup.md

```diff
@@ -0,0 +1,281 @@
+   [Bitcoin Node Setup](https://github.com/babylonlabs-io/covenant-signer/blob/main/docs/deployment.md#2-bitcoind-setup).
+2. Ensure you have access to the private Bitcoin key, also referred to as the 
+   covenant emulator key, which will be used for signing operations.
+<!-- 3 should we include the babylon node setup? -->
+
+## 3. Install covenant emulator binary 
+
+Download [Golang 1.21](https://go.dev/dl) 
+
+Using the go version 1.21. Once installed run: 
+
+```shell
+go version
+```
+
+Subsequently clone the covenant [repository](https://github.com/babylonlabs-io/covenant-emulator).
+
+```shell
+git clone git@github.com:babylonlabs-io/covenant-emulator.git
+cd covenant-emulator
+git checkout <tag>
+```
+
+Run the following command to build the binaries and
+install them to your `$GOPATH/bin` directory:
+
+```shell
+make install
+```
+
+This command will:
+- Build and compile all Go packages
+- Install `covd` binary to `$GOPATH/bin`
+- Make commands globally accessible from your terminal
+
+If your shell cannot find the installed binary, make sure `$GOPATH/bin` is in
+the `$PATH` of your shell. Use the following command to add it to your profile
+depending on your shell.
+
+```shell
+export PATH=$HOME/go/bin:$PATH
+echo 'export PATH=$HOME/go/bin:$PATH' >> ~/.profile
+```
+
+## 4. Setting up the covenant emulator program
+
+### 4.1. Initialize directories
+
+Next, we initialize the node and home directory. It should generate all of the 
+necessary files such as `covd.config`, these files will live in the `<path>` 
+that you set for the `--home`with the below command.
+
+```shell
+covd init --home <path>
+```
+
+After initialization, the home directory will have the following structure:
+
+```shell
+$ ls <path>
+  ├── covd.conf # Covd-specific configuration file.
+  ├── logs      # Covd logs
+```
+
+### 4.2. Configure the covenant emulator
+
+To set up the connection parameters for the Babylon chain and other covenant 
+emulator settings, configure the `covd.conf` file with the following parameters.
+
+```
+# The interval between each query for pending BTC delegations
+QueryInterval = 15s
+
+# The maximum number of delegations that the covd processes each time
+DelegationLimit = 100
+
+# Bitcoin network to run on
+BitcoinNetwork = simnet
+
+# Babylon specific parameters
+
+# Babylon chain ID
+ChainID = bbn-test-5
+
+# Babylon node RPC endpoint
+RPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Babylon node gRPC endpoint
+GRPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Name of the key in the keyring to use for signing transactions
+Key = covenant-key
+
+# Type of keyring to use,
+# supported backends - (os|file|kwallet|pass|test|memory)
+# ref https://docs.cosmos.network/v0.46/run-node/keyring.html#available-backends-for-the-keyring
+KeyringBackend = test
+```
+
+Now we are ready to import the keys from the prior setup.
+
+## 5. Importing your keys from the prior setup
```

Comment:
Hmm section should probably be part of `https://github.com/babylonlabs-io/covenant-emulator/tree/main/covenant-signer` docs